buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt

```diff
@@ -159,6 +159,23 @@ fun Project.createStackDeployTask(profile: String) {
                            |      MINIO_ROOT_USER: admin
                            |      MINIO_ROOT_PASSWORD: adminadmin
                            |
+                           |  minio-startup:
```

Comment:
I thought you will copy it inside the task `startMinio`

but this is even better

Reply:
startMinio -- it;'s gradle task. we still use this name.
but we can just copy a file to S3, need some application which does it (mc).
but also we shouldn't use folder from source code as volume for S3 -- new files are added there